src/helpers/types/models.ts

```diff
@@ -85,6 +85,7 @@ export interface UserAttributes {
     year: number | null;
     children: number | null;
     blocked: boolean;
+    suspect: boolean;
```

Comment:
I'd argue that we should have these states out of the UserAttributes.
UserAttributes reflects the backend interface. Although I think it's better to have it separated, at least now, the same way we have isManager, isBeneficiary, for the sake of simplicity. These UserAttributes are loaded from cache and only updated upon changes on initial authentication.
Although I'd also say that it will need a refactor soon, as more states are going to be loaded.